core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -574,6 +583,11 @@ private[spark] object JsonProtocol {
     SparkListenerExecutorRemoved(time, executorId, reason)
   }
 
+  def logStartFromJson(json: JValue): SparkListenerLogStart = {
+    val version = (json \ "Spark Version").extract[String]
```

Comment:
minor - but maybe call this `sparkVersion` so it's clear that this isn't a version for the logging format (since that doesn't have its own versioning).